spec/features/admin/configuration/tax_rates_spec.rb

```diff
@@ -29,4 +29,37 @@
     click_button "Create"
     expect(page).to have_content("Tax rate has been successfully created!")
   end
+
+  # Adds further CRUD operations: editing, deleting
+  context "while editing" do
+    it "fields can be filled in and dropfdowns retains changes" do
+      visit spree.edit_admin_tax_rate_path(tax_rate.id)
+      fill_in "Rate", with: "0.23"
+      fill_in "Name", with: "GST"
+
+      find(:id, "tax_rate_zone_id").select "Ilhas"
+      find(:id, "tax_rate_tax_category_id").select "Full"
+      click_button "Update"
+      expect(page).to have_content('Tax rate "GST" has been successfully updated!')
+      expect(page).to have_content("0.23")
+    end
+
+    # See #6554: in order to set a Tax Rate as included in the price,
+    # there must be at least one Zone set the "Default Tax Zone"
+    it "checkboxes can be ticked" do
+      visit spree.edit_admin_tax_rate_path(tax_rate.id)
+      uncheck("tax_rate[show_rate_in_label]")
+      check("tax_rate[included_in_price]")
+      click_button "Update"
+      expect(page).to have_content("Included Price Validation")
```

Comment:
This message will have changed now that https://github.com/openfoodfoundation/openfoodnetwork/pull/6545 has been merged, right?

Reply:
Yes, indeed it will - I'll add the correct string here :+1: